app/controllers/search_controller.rb

```diff
@@ -12,7 +12,8 @@ def index
     if search_params.no_search? && params[:format] != "json"
       render action: 'no_search_term' and return
     end
-    search_response = SearchAPI.new(search_params).search
+    variant = search_match_length_variant.variant_name
+    search_response = SearchAPI.new(search_params, match_length_variant: variant).search
```

Comment:
Assuming this param gets passed to Rummager, we should make sure rummager supports it before merging this PR, as it will 400 otherwise.